.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.5.cjs"
```

Comment:
What is this?

Reply:
The correct version of yarn is bundled with the repo (in `.yarn/releases`), as per yarn recommendations. Again, I can remove if you feel like that's an issue.

Reply:
Where can I read more about that recommendation? My initial reaction is that I don't love adding a large file to the repository if we don't need to, but I'd like to understand the argument in favor before deciding.

Reply:
Yeah, the argument is basically "so users don't need to worry about local yarn versions". https://classic.yarnpkg.com/en/docs/cli/policies

Reply:
It makes some sense; let's give it a try.